R/lazyframe__lazy.R

```diff
@@ -265,15 +265,71 @@ LazyFrame_filter = "use_extendr_wrapper"
 
 #' @title New DataFrame from LazyFrame_object$collect()
 #' @description collect DataFrame by lazy query
+#' @param type_coercion Boolean. Do type coercion optimization.
+#' @param predicate_pushdown  Boolean. Do predicate pushdown optimization.
+#' @param projection_pushdown  Boolean. Do projection pushdown optimization.
+#' @param simplify_expression  Boolean. Run simplify expressions optimization.
+#' @param no_optimization  Boolean. Turn off (certain) optimizations.
+#' @param slice_pushdown  Boolean. Slice pushdown optimization.
+#' @param common_subplan_elimination  Boolean. Will try to cache branching subplans that occur on
+#' self-joins or unions.
+#' @param streaming  Boolean. Run parts of the query in a streaming fashion
+#' (this is in an alpha state)
+#' @param collect_in_background Boolean. Detach this query from R session. Computation will start
+#' in background. Get a handle which later can be converted into the resulting DataFrame. Useful
+#' in interactive mode to not lock R session.
+#' @details
+#' Note: use `$fetch()` if you want to run your query on the first `n` rows only.
+#' This can be a huge time saver in debugging queries.
 #' @keywords LazyFrame DataFrame_new
-#' @return collected `DataFrame`
+#' @return collected `DataFrame` or if colkect
```

Comment:
```suggestion
#' @return A `DataFrame`
```
(no idea what you meant after the "or")